rules.md

```diff
@@ -66,3 +66,221 @@ and fail unexpectedly when supplied extra long strings.
 ## H001: Base path can be extracted
 
 If all paths start with the same prefix then it would be cleaner to extract that into the basePath rather than repeating for each path.
+
+##
+
+
+As well as the additional rules coming from Zally, there are also additional Boat Quay rules, some of which are Backbase 
+specific but could be transferable. These additional rules, are documented below.    
+##
+
+## B012: Use Well Understood HTTP Status Codes
+
+Adapted from Zally rule 150. Well-understood codes should be used to ensure that the codes in the spec are being 
+correctly implemented. Any well-understood codes used should be used properly.
+
+### Well-understood codes:
+#### Success Codes
+Code | Method
+-----|-------
+200 | ALL
+201 | POST, PUT
+202 | POST, PUT, DELETE, PATCH
+204 | PUT, DELETE, PATCH
+207 | POST
+
+#### Redirection Codes
+Code | Method
+-----|-------
+301 | ALL
+303 | PATCH, POST, PUT, DELETE
+304 | GET, HEAD
+
+#### Client Side Error Codes
+Code | Method
+-----|-------
+400 | ALL
+401 | ALL
+403 | ALL
+404 | ALL
+405 | ALL
+406 | ALL
+408 | ALL
+409 | POST, PUT, DELETE, PATCH
+410 | ALL
+412 | PUT, DELETE, PATCH
+415 | POST, PUT, DELETE, PATCH
+422 | ALL
+423 | PUT, DELETE, PATCH
+428 | ALL
+429 | ALL
+
+#### Server Side Error Codes
+Code | Method
+-----|-------
+    500 | ALL
+    501 | ALL
+    502 | ALL
+    503 | ALL
+    504 | ALL
+
+## B011: Use standard HTTP status codes
+
+Adapted from Zally rule 150, this adds extra HTTP codes to the list of standard and well-understood codes. Any HTTP codes 
```

Comment:
Maybe clearer as: This rule highlights when a HTTP code is standard but not well-understood. Standard codes can be used, but it's best practice to use well-understood codes instead.